### PR TITLE
Migrate away from Travis ci

### LINKED ISSUE #22: Migrate completely from Travis ci
## 🚀 Feature Proposal

Migrate completely from Travis CI

## Motivation

Since we have just added GitHub actions I think we can now move away of Travis ci completely

### PR DESCRIPTION
**Pull request by veritem**
fix #22

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by makuzaverite** [comment]
I'm not sure why tests are failing for these actions

**Comment by mcollina** [comment]
could you fuix them?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,13 +13,15 @@ jobs:
...
-        - 9092:9092
+  #   services:
+  #     kafka:
+  #       image: confluentinc/cp-kafka
+  #       ports:
+  #       - 9092:9092
     steps:
     - uses: actions/checkout@v2
+    - name: Build the stack
+        run: docker-compose up -d
```
Comment (StarpTech) [inline .github/workflows/ci.yml]: I think this is longer necessary when we start kafka as a service.

**Comment by makuzaverite** [comment]
Anyone who knows whats wrong with running docker compose in a workofw

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,21 +12,19 @@ jobs:
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x]
-    services:
-      kafka:
-        image: confluentinc/cp-kafka
-        ports:
-        - 9092:9092
     steps:
-    - uses: actions/checkout@v2
```
Comment (Fdawgs) [inline .github/workflows/ci.yml:22]: checkout needs to be left in or else the workflow won't be able to access the repository or its contents

**Comment by makuzaverite** [comment]
@Fdawgs  still not working is there anything I'm doing wrong??

**Comment by Fdawgs** [comment]
> 
> 
> @Fdawgs still not working is there anything I'm doing wrong??

Sorry, probably didn't explain that well.
`- uses: actions/checkout@v2` needs to not be commented out.

**Comment by Eomm** [comment]
fixed https://github.com/fastify/fastify-kafka/pull/21

**Pull request closed without merging**